### ISSUE TITLE
How to define mocks

### ISSUE THREAD

**Issue by tomitrescak** [MadRabbit/graphql-mock#18]
Guys, I have just spent 2 hours trying to figure out how to define mocks.
You mention the `standard way`.

The standard mosck is following:

```
const mocks = [
  {
    request: {
      query: QUERY,
      variables: {
        from: 1,
        to: 10
      },
    },
    result: {
      data: {
        notifications: [{ id: '1', name: 'Buck', breed: 'bulldog' }],
      },
    },
  },
];
```

I keep getting all kinds of errors. Would you be so kind and help a pal out with otherwise excellent plugin? You do not have a single example with a mock .. or maybe I'm blind.

**Comment by tomitrescak** [comment]
Even without mocks this does not seem to work. For any query that works well on client I am receiving following type of errors:

```
Unexpected GraphQL request:
query notifications($start: Int!, $end: Int!) {
  notifications(start: $start, end: $end) {
    id
    code
  }
}

VARIABLES:
{
  "start": 0,
  "end": 10
}
```

**Comment by tomitrescak** [comment]
OK, I solved it using the proposed

```
graphqlMock.expect(query).reply({ // <- `query` from the code above
      items: [
        { id: '1', name: 'one' },
        { id: '2', name: 'two' }
      ]
    });
```

FYI: That was very confusing. In the main page you mention to define mocks using the "standard way" passed into mockClient constructor. To this moment I do not know how to do this.


The notation `expect(query).reply` looks like the expectation that the query will reply with that given data. That's why I got stuck in a loop of trying to define mocks in the constructor and then "test" it via `reply` expectation.
